.travis.yml

```diff
@@ -19,7 +19,7 @@ services: docker
 
 env:
   matrix:
-  - INSTANCE=default-debian-7
+  # - INSTANCE=default-debian-7
```

Comment:
Any particular reason why you are commenting out debian-7 and centos-6? We like to have the tests running to expose any problems